ui/helpers/constants/zendesk-url.js

```diff
@@ -49,6 +49,8 @@ const ZENDESK_URLS = {
     'https://support.metamask.io/networks-and-sidechains/managing-networks/verifying-custom-network-information/',
   NETWORK_DEPRECATED:
     'https://support.metamask.io/networks-and-sidechains/eth-on-testnets/',
+  SMART_TRANSACTIONS_LEARN_MORE:
```

Comment:
We already have this constant, can you use that one? https://github.com/MetaMask/metamask-extension/blob/main/shared/constants/smartTransactions.ts#L30

There are 2 reasons why it's in `shared/constants/smartTransactions.ts`:
- To keep smart transactions constants in one place
- It shouldn't be in zendesk-url.js since it's not a zendesk URL

Reply:
Yeah this was an oversight. Thanks for the heads up